tests/test_create_config.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+import subprocess
+
+
+def test_create_config():
+    result = subprocess.run(["python3.7", "-m", "universum", "create-config"], capture_output=True, check=True)
+    new_command = result.stdout.splitlines()[3]
```

Comment:
This approach to extracting the command-line from the output is too rigid.
It creates the hidden dependency between the output and the test.
I propose to add some marker, like natural dollar sign before the printed command-line and base the extraction on the marker.